Frontend/RecRock-Calendar-Frontend/src/app/calendar/components/login/login.component.ts

```diff
@@ -31,25 +32,24 @@ export class LoginComponent implements OnInit {
   ngOnInit(): void {}
 
   login(): void {
+    let userName = this.loginForm.value['usernameFormControl'];
+    let userPassword = this.loginForm.value['passwordFormControl'];
     if (this.loginForm.valid) {
-      this.userService
-        .login(
-          this.loginForm.value['usernameFormControl'],
-          this.loginForm.value['passwordFormControl']
-        )
-        .then((user) => {
-          if (user) {
-            if (this.rememberMe) {
-              localStorage.setItem(StorageConstants.userId, user.id);
-              localStorage.setItem(StorageConstants.userFullName, user.fullName);
-            }
-            sessionStorage.setItem(StorageConstants.userId, user.id);
-            sessionStorage.setItem(StorageConstants.userFullName, user.fullName);
-            this.router.navigate(['/calendar']);
+      this.userService.login(userName, userPassword).then((user) => {
+        if (user) {
+          if (this.rememberMe) {
+            localStorage.setItem(StorageConstants.userId, user.id);
```

Comment:
I would create a helper for the local storage and for the session storage in which we two methods, for example setCurrentUser and clearCurrentUser, this way the logic will be much cleaner